# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1264039109 18000
#      Wed Jan 20 20:58:29 2010 -0500
# Node ID 085d4e0251554cd9522705c406bf65317aa8cdad
# Parent  a822560a3ce347f6ef9dc1abb5aeeb2ef5f26535
libcruft/misc/quit.cc: untabify

diff --git a/libcruft/ChangeLog b/libcruft/ChangeLog
--- a/libcruft/ChangeLog
+++ b/libcruft/ChangeLog
@@ -1,8 +1,12 @@
+2010-01-20  John W. Eaton  <jwe@octave.org>
+
+	* misc/quit.cc: Untabify.
+
 2010-01-20  John W. Eaton  <jwe@octave.org>
 
 	* Makefile.am (EXTRA_DIST): Remove STOP.patch from the list.
 
 2010-01-20  Jaroslav Hajek  <highegg@gmail.com>
 
 	* misc/quit.h: Define inline-function variant of OCTAVE_QUIT in C++
 	mode.
diff --git a/libcruft/misc/quit.cc b/libcruft/misc/quit.cc
--- a/libcruft/misc/quit.cc
+++ b/libcruft/misc/quit.cc
@@ -84,22 +84,22 @@ octave_rethrow_exception (void)
   if (octave_interrupt_state)
     {
       octave_interrupt_state = -1;
       octave_throw_interrupt_exception ();
     }
   else
     {
       switch (octave_exception_state)
-	{
-	case octave_exec_exception:
-	  octave_throw_execution_exception ();
-	  break;
+        {
+        case octave_exec_exception:
+          octave_throw_execution_exception ();
+          break;
 
-	case octave_alloc_exception:
-	  octave_throw_bad_alloc ();
-	  break;
+        case octave_alloc_exception:
+          octave_throw_bad_alloc ();
+          break;
 
-	default:
-	  break;
-	}
+        default:
+          break;
+        }
     }
 }
